<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Emily Kahl</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1454</width>
    <height>727</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QtNEMD</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="PlotWidget" name="plot_window"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="temperature_label">
          <property name="text">
           <string>Temperature = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="volume_label">
          <property name="text">
           <string>Volume = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="npart_label">
          <property name="text">
           <string>N particles = </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="start_button">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pause_button">
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="restart_button">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="nemd_checkbox">
          <property name="text">
           <string>Enable NEMD</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="field_spinbox">
            <property name="minimumSize">
             <size>
              <width>129</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Field strength</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>Spin box input for field strength</string>
            </property>
            <property name="accessibleName">
             <string>Field strength</string>
            </property>
            <property name="accessibleDescription">
             <string>Spin box input for field strength</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="field_label">
            <property name="text">
             <string>Field strength</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="lj_eps_spinbox">
            <property name="minimumSize">
             <size>
              <width>129</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>LJ strength</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>Spin box input for LJ repulsion strength</string>
            </property>
            <property name="accessibleName">
             <string>LJ strength</string>
            </property>
            <property name="accessibleDescription">
             <string>Spin box input for LJ repulsion strength</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lj_eps_label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;WCA Îµ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="density_spinbox">
            <property name="minimumSize">
             <size>
              <width>129</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="density_label">
            <property name="text">
             <string>Fluid density</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="temp_spinbox">
            <property name="minimumSize">
             <size>
              <width>129</width>
              <height>0</height>
             </size>
            </property>
            <property name="statusTip">
             <string>Set the temperature</string>
            </property>
            <property name="whatsThis">
             <string>Spinbox to set the temperature of the simulation</string>
            </property>
            <property name="accessibleName">
             <string>Temperature control</string>
            </property>
            <property name="accessibleDescription">
             <string>Spinbox to set the temperature of the simulation (can only be used while the smulation is stopped)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="tr_label">
            <property name="text">
             <string>Temperature</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;g(r) radial distribution function&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PlotWidget" name="rdf_window"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="textbrowser_label">
        <property name="text">
         <string>Fortran Input file (read only)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="input_textbrowser">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>271</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Input parameters for simulation</string>
        </property>
        <property name="whatsThis">
         <string>Input parameters for simulation in input file format</string>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1454</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open_input_file"/>
    <addaction name="save_input_file"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="plot_action">
   <property name="text">
    <string>New real-time plot</string>
   </property>
   <property name="toolTip">
    <string>Plotting dialog</string>
   </property>
   <property name="whatsThis">
    <string>Select quantities to plot in real time</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="open_input_file">
   <property name="text">
    <string>Read parameters from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="save_input_file">
   <property name="text">
    <string>Save parameters to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStructure_plots">
   <property name="text">
    <string>Structure plots</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
